docs/scripts/util/crawlAllLinksFromAllPages.ts

```diff
@@ -0,0 +1,41 @@
+import puppeteer from 'puppeteer';
+import { runArrayPromiseInOrder } from '.';
+
+export async function crawlAllLinksFromAllPages(allPages: string[]) {
+  type AllPagesPaths = Map<
+    string,
+    {
+      pageUrl: string;
+      links: { href: string; tagName: string; tagText: string }[];
+    }
+  >;
```

Comment:
For readability, types should not be declared within functions

Reply:
Done 77db0129b802808925840a8fc92f3c192acfce35